api/server/server.go

```diff
@@ -116,9 +116,24 @@ const (
 	// EnvLBPlacementAlg is the algorithm to place fn calls to fn runners in lb.[0w
 	EnvLBPlacementAlg = "FN_PLACER"
 
-	// EnvMaxRequestSize sets the limit in bytes for any API request's length.
+	// EnvMaxRequestSize sets the limit in bytes for any API request body's length.
 	EnvMaxRequestSize = "FN_MAX_REQUEST_SIZE"
 
+	// EnvMaxHeaderSize sets the limit in bytes for any API request body's length.
+	EnvMaxHeaderSize = "FN_MAX_REQUEST_HEADER_SIZE"
+
+	// EnvReadTimeout sets the timeout limit for reading a http request body.
+	EnvReadTimeout = "FN_REQUEST_BODY_READ_TIMEOUT"
```

Comment:
if these timeouts have to be in seconds, can you please rename or document that?